src/main/scala/gw/cda/api/outputwriter/JdbcOutputWriter.scala

```diff
@@ -36,11 +36,14 @@ import org.apache.spark.sql.functions.col
 import org.apache.spark.sql.jdbc.JdbcDialects
 import org.apache.spark.sql.types.StructType
 
-
 private[outputwriter] class JdbcOutputWriter(override val outputPath: String, override val includeColumnNames: Boolean,
                                              override val saveAsSingleFile: Boolean, override val saveIntoTimestampDirectory: Boolean,
                                              override val clientConfig: ClientConfig) extends OutputWriter {
 
+  val tableDotColumnValues: Array[String] = Option(clientConfig.outputSettings.largeTextFields).getOrElse("").replace(" ", "").split(",")
+
+  private[outputwriter] val tableColumnValues:Array[String] = tableDotColumnValues
```

Comment:
```suggestion
  private[outputwriter] val tableColumnValues: Set[String] = Option(clientConfig.outputSettings.largeTextFields).getOrElse("").replace(" ", "").split(",").toSet
```
You're checking whether an element is contained within it, make it a set.

Reply:
Also, I would rename to something that makes the purpose apparent. I don't know what `tableColumnValues` is supposed to refer to.